src/components/socials/Social.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Social = () => {
+  return (
+    <div className="relative top-[101px] flex h-[236px] w-full overflow-hidden">
```

Comment:
```suggestion
    <div className="relative flex h-[236px] w-full overflow-hidden">
```
you can have it lie at the top of the screen for now, since the navbar should appear above it by default